3_javascript/javascript1/week1/Startup name generator.js

```diff
@@ -0,0 +1,26 @@
+let firstWords=["Innovate",
+"Nexa",
+"Fusion",
+"Tech",
+"Vivid",
+"Spark",
+"Strive",
+"Elevate",
+"Shift",
+"Nimbex"];
+let secondWords=["Hub",
+"Nova",
+"Wave",
+"Zenith",
+"Synergy",
+"Loom",
+"Core",
+"Nova",
+"Edge",
+"Ventures"
+];
+
+const startupName = firstWords.map((word, index) => word + " " + secondWords[index]);
+console.log(startupName);
+const totalCharacters = startupName.map(name => name.length);
+console.log("The startup: " + startupName[0] + " " + "contains" + " "+ totalCharacters[0] + " " + "characters");
```

Comment:
💡 Consider using `const` for those variables you don't update, like `firstWords` and `secondWords`
👏 Nice approach in mapping firstWords and secondWords to create startup names.
🚩 However, for true randomness as per the initial instructions, think about incorporating `Math.random` as mentioned in the question.

Reply:
Thankyou for pointing out the suggestions. I updated the changes now.